src/labkey/constants.ts

```diff
@@ -14,40 +14,90 @@
  * limitations under the License.
  */
 export interface Container {
+    activeModules: Array<string>
+    folderType: string
+    formats: {
+        dateFormat: string
+        dateTimeFormat: string
+        numberFormat: string
+    }
+    hasRestrictedActiveModule: boolean
+    iconHref: string
+    id: string
+    isContainerTab: boolean
+    isWorkbook: boolean
+    name: string
+    parentId: string
+    parentPath: string
     path: string
+    sortOrder: number
+    startUrl: string
+    title: string
+    type: string
 }
 
-export interface ExperimentalFeatures {
-    containerRelativeURL: boolean
+export enum ExperimentalFeatures {
+    containerRelativeURL = 'containerRelativeURL',
+    disableGuestAccount = 'disableGuestAccount',
+    javascriptErrorServerLogging = 'javascriptErrorServerLogging',
+    javascriptMothership = 'javascriptMothership',
+    useExperimentalCoreUI = 'useExperimentalCoreUI',
+    strictReturnUrl = 'strictReturnUrl',
```

Comment:
Was this just a subset of the flags based on what was needed in the Biologics or other apps? Just curious.

Reply:
This is the set of flags as currently serialized in `PageFlowUtil`s `jsInitObject` method. I've added a note to keep this updated [here](https://github.com/LabKey/platform/blob/39ad613cb984656744830b7bd7f3176ca6ea796e/api/src/org/labkey/api/util/PageFlowUtil.java#L2109).